app/src/main/java/ca/sfu/pacmacro/PlayerActivity.java

```diff
@@ -249,9 +254,15 @@ AppCompatDialog createGpsAlertDialog() {
         AlertDialog.Builder dialogBuilder = new AlertDialog.Builder(PlayerActivity.this, R.style.Theme_AppCompat_Dialog);
         dialogBuilder.setTitle(getApplicationContext().getString(R.string.warning));
         dialogBuilder.setMessage(getApplicationContext().getString(R.string.gps_alert_dialog_msg));
-        dialogBuilder.setCancelable(false);
+        dialogBuilder.setCancelable(true);
```

Comment:
Can you set this back to `false`, so that the user MUST read the dialog and interact with the buttons?